blank_rust_project/README.md

```diff
@@ -0,0 +1,76 @@
+<br />
```

Comment:
Please add instructions on how to run tests.

Please also add instructions on how to run it with local node.

Reply:
done (linked to an existing docs tutorial for local node, since it's an involved process)